suite-native/transactions/src/components/TransactionDetail/TransactionDetailInputsSheet.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Icon } from '@trezor/icons';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { Box, Card, Text, VStack } from '@suite-native/atoms';
+import { useFormatters } from '@suite-common/formatters';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+
+import { TransactionDetailSheet } from './TransactionDetailSheet';
+
+type TransactionDetailInputsSheetProps = {
+    isVisible: boolean;
+    transaction: WalletAccountTransaction;
+    onSheetVisibilityChange: () => void;
+};
+
+type TransactionAddressAmountProps = {
+    address: string;
+    amount: string;
+};
+
+const addressAmountColumnStyle = prepareNativeStyle(_ => ({
+    maxWidth: '42.5%',
+}));
+
+const TransactionAddressAmount = ({ address, amount }: TransactionAddressAmountProps) => (
+    <Box>
+        <Text variant="hint" color="gray1000" numberOfLines={1} ellipsizeMode="middle">
+            {address}
+        </Text>
+        <Text variant="label" color="gray600">
+            {amount}
+        </Text>
+    </Box>
+);
+
+export const TransactionDetailInputsSheet = ({
+    isVisible,
+    onSheetVisibilityChange,
+    transaction,
+}: TransactionDetailInputsSheetProps) => {
+    const { applyStyle } = useNativeStyles();
+    const { CryptoAmountFormatter } = useFormatters();
+
+    const txInputs = transaction.details.vin;
+    const txOutputs = transaction.details.vin;
+
+    const formatCryptoInputAmount = (amount?: string) =>
```

Comment:
Please never use both `formatNetworkAmount` and formatters together. Basically never use `formatNetworkAmount`, I would say that is deprecated for us. 

This should work exactly same:
```tsx
CryptoAmountFormatter.format(amount, { symbol: transaction.symbol });
```

Also please remove this `formatCryptoInputAmount` and move formatter in place where you need it, in you case to `TransactionAddressAmount` component.

Then you can use it directly as component:

```tsx
<CryptoAmountFormatter value={value} symbol={symbol} />
```

Reply:
commented as TODO in https://github.com/trezor/trezor-suite/commit/595577f78b4355315f8b8ab716104ed58e366b10